src/App.js

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 
 import './App.css';
+import Calculator from './components/caculator';
 
 const App = () => (
```

Comment:
- [ ] Equally, you will need to make this component class-based to fulfill the requirements of this project and move a step toward approval.

Reply:
I tried but I'm getting an error from react saying Component should be written as a pure function  react/prefer-stateless-function

Reply:
@IshmaelKargbo . Read about pure component and make use of it.